src/access_control.ruleset

```diff
@@ -6,32 +6,32 @@
 
 using namespace gaia::access_control;
 
-// Here we define our declarative rules
-//
+// Here we define our declarative rules.
 ruleset access_control_ruleset
 {
+    // A new scan is added.
+    //
+    // Reacts to:
+    //      new scan record
```

Comment:
This appears to be a different format than the following reacts to items.

Reply:
Of course it's a different format - this is the only `on_insert` rule in the whole ruleset. The other rules react to updated records or even updated record fields, not newly inserted records.

Reply:
I might change it to be explicit:
```suggestion
    //      A new row in the scan table.
```